packages/fether-electron/src/main/app/options/config/index.js

```diff
@@ -0,0 +1,82 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import path from 'path';
+import url from 'url';
+
+import staticPath from '../../utils/staticPath';
+
+const INDEX_HTML_PATH =
+  process.env.ELECTRON_START_URL ||
+  url.format({
+    pathname: path.join(staticPath, 'build', 'index.html'),
+    protocol: 'file:',
+    slashes: true
+  });
+
+// Icon path differs when started with `yarn electron` or `yarn start`
+let iconPath = path.join(staticPath, 'assets', 'icons', 'icon.png');
+let iconDockPath = '';
+
+if (process.platform === 'win32') {
+  iconPath = path.join(staticPath, 'assets', 'icons', 'win', 'icon.ico');
+} else if (process.platform === 'darwin') {
+  // https://github.com/electron/electron/blob/master/docs/api/native-image.md#template-image
+  iconPath = path.join(
+    staticPath,
+    'assets',
+    'icons',
+    'mac',
+    'iconTemplate.png'
+  );
+  iconDockPath = path.join(
+    staticPath,
+    'assets',
+    'icons',
+    'mac',
+    'iconDock.png'
+  );
+}
+
+const shouldUseDevTools = process.env.NODE_ENV !== 'production';
+const shouldUseFrame = process.platform === 'win32';
+
+const windowPosition =
+  process.platform === 'win32' ? 'trayBottomCenter' : 'trayCenter';
+
+// API docs: https://electronjs.org/docs/api/browser-window
+const DEFAULT_OPTIONS = {
+  alwaysOnTop: true,
+  dir: staticPath,
+  frame: true,
+  height: 640,
+  hasShadow: true,
+  icon: iconPath,
+  iconDock: iconDockPath,
+  index: INDEX_HTML_PATH,
+  resizable: false,
+  show: false, // Run showWindow later
+  showDockIcon: true, // macOS usage only
+  tabbingIdentifier: 'parity',
+  webPreferences: {
+    devTools: shouldUseDevTools, // Security
```

Comment:
Why is this a security risk? It helps a lot when users post their console logs, even in production

Reply:
I've now allowed dev tools. It's not mentioned at the following links:
* https://electronjs.org/docs/tutorial/security

Note that I have not focussed on improving security in this PR. I was going to create a separate PR for security-specific aspects as a result of going through https://github.com/paritytech/fether/issues/124.

I created a separate branch a while ago where I started going through security-related aspects https://github.com/paritytech/fether/commit/4f6c2fd6fabeea7972e5b7d26e1715e2b9944008

Would you like that to remain as a separate PR?

Reply:
Yes, this PR is big enough, so let's do another security-focused PR